config/puma.rb

```diff
@@ -0,0 +1,13 @@
+workers Integer(ENV['WEB_CONCURRENCY'] || 2)
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.